### DISCUSSION TITLE
Setup nvim-tree ui

### DISCUSSION THREAD

**Discussion by DinkyTrady** [nvim-tree/nvim-tree.lua#1630] [Q&A] [answered]
How i set up neovim-tree like https://github.com/kyazdani42/nvim-tree.lua#screenshots
I don't know how to do

**Comment by alex-courtis** [comment, marked as answer]
Some of those are quite old and we don't have the setup source.

Which particular feature are you looking for?

**Reply by DinkyTrady** [reply]
Change color

**Reply by alex-courtis** [reply]
`:help nvim-tree-highlight`

You only need to override that which you wish to change.
